README.md

```diff
@@ -1,6 +1,6 @@
 # 📣 Tracker card
 
-A card to track updates for [custom cards](https://github.com/custom-components/custom_cards) and [custom components](https://github.com/custom-components/custom_components). It uses custom_components to sync versions and allows for **automatic updates of cards**. ♠️♣️♥️♦️
+A card to track updates for [custom cards](https://github.com/custom-cards) and [custom components](https://github.com/custom-components). It uses a [custom_component](https://github.com/custom-components/custom_updater) to sync versions and allows for **updates of cards and components from the UI**. ♠️♣️♥️♦️
```

Comment:
Maybe " ... It uses the [custom_updater]](https://github.com/custom-components/custom_updater) component to ..."